### ISSUE TITLE
[CoE Starter Kit - BUG] ISSUE TITLE

### ISSUE THREAD

**Issue by KevinMathis** [microsoft/coe-starter-kit#2931]
### Describe the issue

I'm setting up the CoE Starter kit Power BI Report and am noticing many apps and cloud flows that have the Owner/Creator values set to SYSTEM or NULL. 

Is there a reason these components are not showing a user as the owner? 
How can I reset these records to show the true component owner?

For apps, the admin_appownerdisplayname value is often either blank or 'SYSTEM'.
For cloud flows, the admin_flowcreator value is often blank.

Thanks for the help!

![apps and flows with no owner](https://user-images.githubusercontent.com/69484878/172255680-5150ff9f-5bb7-465d-89fe-a366defdc04c.png)

### Expected Behavior

I would expect the majority of these components to have Owner/Creator values with names of Power Apps or AAD users.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.38

### What app or flow are you having the issue with?

CoE Starter Kit Power BI Report

### Steps To Reproduce

1. Publish Power BI Report
2. Open Power BI Report
3. Navigate to the Overview - Power Apps report tab
4. view the "Top App Maker" visual / list
5. top 2 "app makers" are 'blank' and 'SYSTEM'

**Comment by Jenefer-Monroe** [comment]
Can you please check a few things for me
1. Are all the [inventory flows]( https://docs.microsoft.com/power-platform/guidance/coe/setup-core-components) turned on? 
You can check this in the [Admin – Command Center]( https://docs.microsoft.com/power-platform/guidance/coe/core-components#admin---command-center) by going CoE Flows > Inventory > make sure all flows show as on.
![image](https://user-images.githubusercontent.com/89584709/172395138-e823ba0f-e7e2-4430-9de4-fe93993701ff.png)

2. Are there inventory flow failures? 
You can check this in the same [Admin – Command Center]( https://docs.microsoft.com/power-platform/guidance/coe/core-components#admin---command-center) app by going to Service Health and seeing if there are failures.
![image](https://user-images.githubusercontent.com/89584709/172395207-fed5bdda-ba54-4fc0-a050-4a2362766244.png)

3. Does the data show as you expect in the [Power Platform Admin View]( https://docs.microsoft.com/power-platform/guidance/coe/core-components#power-platform-admin-view) app? 
![image](https://user-images.githubusercontent.com/89584709/172395280-11a99b41-6201-43f3-b06a-bc95393ac816.png)
 
4. Please give the report name you are using in the pictures above. What file from the download, and what report page are you on? 

Thanks for using CoE

**Comment by KevinMathis** [comment]
Here's a picture of this issue in a standard report page.
![image](https://user-images.githubusercontent.com/69484878/172693595-88f23204-dd65-480b-a429-f8e047c53069.png)

I will try the full inventory run and let you know the result. Thanks!

**Comment by KevinMathis** [comment]
The full inventory run did not resolve this issue. With further investigation we have noticed two common scenarios that seem to be contributing to this issue:

Are there tools in the CoE starter kit to clean up inventory information for the following scenarios?

1. Apps that appear to have been deleted
2. Apps that have owners with no "Maker" record and an inactive "User" record

**Comment by Jenefer-Monroe** [comment]
Yes these two flows do the work to clean up these objects. Please be sure they are turned on and running.

- CLEANUP - Admin | Sync Template v3 (Check Deleted)
- CLEANUP - Admin | Sync Template v3 (Orphaned Makers)

Blank is shown for orphaned objects in some reports, so it could just be that you have a lot of these. 
Orphaned objects can be cleaned up by the admin persona in the [Set App Permissions](https://docs.microsoft.com/power-platform/guidance/coe/core-components#set-app-permissions) and [Set Flow Permissions](https://docs.microsoft.com/power-platform/guidance/coe/core-components#set-flow-permissions) apps. 
And we have an entire area of investment for you to help with [Cleanup for orphaned resources](https://docs.microsoft.com/power-platform/guidance/coe/governance-components#cleanup-for-orphaned-resources) 

I would suggest you please 
1. Ensure the two clean up flows mentioned at the top are turned on, running, and passing
2. Using your Power Platform Admin View app see how many orphaned apps you have in inventory
![image](https://user-images.githubusercontent.com/89584709/173983576-f7a4a072-fe56-4d57-af46-accae73c87e9.png)
